modules/aggs-matrix-stats/build.gradle

```diff
@@ -17,6 +17,7 @@
  * under the License.
  */
 apply plugin: 'elasticsearch.rest-resources'
+apply plugin: 'elasticsearch.yaml-rest-test'
```

Comment:
doesn't yaml-rest-test apply the rest-resources plugin?

Reply:
It does, wasn't sure if we wanted to keep it explicit since most projects are configured in some way or if we should just let it be implicit via the yaml-rest-test. 

I don't have a strong opinion either way.

Reply:
I don't think we need to be explicit. We are saying this is a "yaml test project", we should assume that plugin will apply any dependencies.